Index: profiles/ipwa/modules/contrib/addtocalendar/addtocalendar.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- profiles/ipwa/modules/contrib/addtocalendar/addtocalendar.module	(revision 561fe485d81988368495659a747b3d6685ba16eb)
+++ profiles/ipwa/modules/contrib/addtocalendar/addtocalendar.module	(revision )
@@ -189,8 +189,18 @@
       $build['addtocalendar'] = [];
       $settings = $variables['addtocalendar_settings'];
       if ($formatter_settings['addtocalendar_show']) {
+
         $timeZone = date_default_timezone_get();
-        $date = format_date(strtotime($variables['element']['#object']->field_date['und'][0]['value']), 'custom', 'm/d/Y g:ia');
+        if (!empty($variables['element']['#object']->field_event_datum)) {
+          $occurence = count($variables['element']['#object']->field_event_datum['und']);
+          if ($occurence > 1) {
+            $date = format_date(strtotime($variables['element']['#object']->field_event_datum['und'][0]['value']), 'custom', 'm/d/Y g:ia');
+            $enddate = format_date(strtotime($variables['element']['#object']->field_event_datum['und'][$occurence - 1]['value2']), 'custom', 'm/d/Y g:ia');
+          }else{
+            $date = format_date(strtotime($variables['element']['#object']->field_event_datum['und'][0]['value']), 'custom', 'm/d/Y g:ia');
+            $enddate = format_date(strtotime($variables['element']['#object']->field_event_datum['und'][0]['value2']), 'custom', 'm/d/Y g:ia');
+          }
+        }
         $build['addtocalendar']['atc_date_start'] = array(
           '#type' => 'html_tag',
           '#tag' => 'var',
@@ -236,7 +246,7 @@
           );
 
         }
-        $build['addtocalendar']['atc_date_end']['#value'] = format_date(strtotime($build['addtocalendar']['atc_date_end']['#value']), 'custom', 'm/d/Y g:ia');
+        $build['addtocalendar']['atc_date_end']['#value'] = $enddate;
 
         $build['addtocalendar']['atc_timezone'] = array(
           '#type' => 'html_tag',
